src/store/cutlery/cutlerySlice.tsx

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+interface CutleryState {
+  addCutlery: boolean;
+}
+
+const initialState: CutleryState = {
+  addCutlery: false,
+};
+
+export const cutlerySlice = createSlice({
```

Comment:
for the pet project purposes it's fine, but in a real life probably a seperate slice for holding a boolean value is too much 😀

Reply:
It will be better to keep a whole final section in one slice? (name: string, addCutlery: boolean, addNapkins: boolean). It can be inconsistent. Every record has its own slice and then whole section in one slice.

Reply:
as I said, for this project it's fine, it can stay like this, but probably in a bigger app the state could be redesigned to keep everything in one/two slices